packages/one-app-bundler/__tests__/utils/extendWebpackConfig.spec.js

```diff
@@ -11,11 +11,16 @@
  * or implied. See the License for the specific language governing permissions and limitations
  * under the License.
  */
+const webpack = require('webpack');
+const HolocronModuleRegisterPlugin = require('holocron-module-register-webpack-plugin');
 
 const extendWebpackConfig = require('../../utils/extendWebpackConfig');
 const getConfigOptions = require('../../utils/getConfigOptions');
 const getCliOptions = require('../../utils/getCliOptions');
 
+const mockHolocronModuleRegisterPlugin = () => new HolocronModuleRegisterPlugin('my-new-holocron-module');
+const mockWebpackDefinePlugin = () => new webpack.DefinePlugin({ 'global.BROWSER': JSON.stringify(true) });
```

Comment:
i think to make it clearer this will replace the existing plugin it should be something like "overridden"

Reply:
Done! I went with the word "Overriding" as it describes that this plugin is overriding the existing version.